crypto/ct/ct_sct.c

```diff
@@ -223,6 +237,54 @@ int SCT_set1_signature(SCT *sct, const unsigned char *sig, size_t sig_len)
     return 1;
 }
 
+int SCT_set1_serialized_signature(SCT *sct, const unsigned char **in,
```

Comment:
I meant: the signature is either in the SCT, in which case you presumably don't need a public API method to parse it separately, or the signature is a standalone blob, in which case you know the total length of the blob, and you want a method that reads either the entire blob or fails. (The "advance the pointer" methods are prone to ignoring trailing garbage.)

Anyway, too late to change now. (A CT_SIGNATURE probably isn't appropriate because the struct is the TLS digitally-signed, which is more general. But we can always consider adding that later.)